munimap/application.py

```diff
@@ -77,17 +77,18 @@ def default(self, o):
 
     configure_extensions(app)
 
-    project_static_folder = os.path.join(app.config.get('PROJECT_DIR'), 'static')
-
-    # overwrite application static function
-    # now it searchs first in static folder of current project
-    # if requested file is not found, it's delevired
-    # from application static folder
+    # Overwrite application static function
+    # now it searches first in default static folder.
+    # If requested file is not found there, it will be delivered
+    # from custom static folder. This ensures that files
+    # in default static folder cannot be overwritten.
     def _static(filename):
         folder = app.static_folder
-        project_static_file = os.path.join(project_static_folder, filename)
-        if os.path.exists(project_static_file):
-            folder = project_static_folder
+        default_static_file = os.path.join(folder, filename)
+        if not os.path.exists(default_static_file):
+            custom_static_folder = app.config.get('CUSTOM_STATIC_DIR')
+            if custom_static_folder is not None:
+                folder = custom_static_folder
```

Comment:
this should work?
```suggestion
            folder = app.config.get('CUSTOM_STATIC_DIR', folder)
```

Reply:
see https://github.com/stadt-bielefeld/bielefeldGEOCLIENT/blob/8d34a6182d2c170da72354256a5888799bf98d2c/munimap/application.py#L414

Reply:
Good Point. Works :+1: